package.json

```diff
@@ -8,6 +8,8 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "sweetalert2": "^11.4.26",
```

Comment:
This file should be  commit when changes. Note that dependencies should not be committed.
Please undo this commit